app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/search/SearchNavigation.kt

```diff
@@ -122,6 +123,7 @@ private fun SearchType.toTypeString(): String =
         SearchType.Vault.SecureNotes -> SEARCH_TYPE_VAULT_SECURE_NOTES
```

Comment:
`determineSearchType` will crash with the current config, I think you just need this:
```kotlin
   SEARCH_TYPE_VAULT_SSH_KEYS -> SearchType.Vault.SshKeys
```